core/retail-basket.tab.info

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "retail-basket",
+    "description": "This is a transnational data set which contains all the transactions occurring between 01/12/2010 and 09/12/2011 for a UK-based and registered non-store online retail. The company mainly sells unique all-occasion gifts. Many customers of the company are wholesalers.",
```

Comment:
Is that 1st of Dec to 9th of Dec or Jan 12th to Sep 12th. It is a UK data set so maybe the first, but US users will see it differently.
I vote for the scientific version of using the ISO format!